src/Evolution/Systems/CurvedScalarWave/Tags.hpp

```diff
@@ -40,4 +41,23 @@ struct ConstraintGamma2 : db::SimpleTag {
   using type = Scalar<DataVector>;
   static std::string name() noexcept { return "ConstraintGamma2"; }
 };
+
+/*!
+ * \brief Tags corresponding to various constraints of the scalar
+ * wave system, and their diagnostically useful combinations.
+ * \details For details on how these are defined and computed, see
+ * `OneIndexConstraintCompute`, `TwoIndexConstraintCompute`
+ */
+template <size_t Dim>
```

Comment:
Dim -> SpatialDim, here and throughout PR

Reply:
changed everywhere (I think !)